packages/block-library/src/query/edit/inspector-controls/sticky-control.js

```diff
@@ -8,6 +8,7 @@ const stickyOptions = [
 	{ label: __( 'Include' ), value: '' },
 	{ label: __( 'Exclude' ), value: 'exclude' },
 	{ label: __( 'Only' ), value: 'only' },
+	{ label: __( 'Ignore' ), value: 'ignore' },
```

Comment:
`Ignore` label is not self explanatory in this context and is quite similar to the `exclude`, where it's `ignore post stickiness`.

Maybe a different control can be used with help texts? --cc @jameskoster

Reply:
Perhaps 'Ignore stickiness' is okay for now?

Reply:
Ignore stickiness 😂